shared/panels/issue-creation-panel/_features/defintion-form/defintion-form.tsx

```
@@ -11,6 +11,7 @@ import { Textarea } from "@/shared/ui/textarea";
 import { TypographyMuted } from "@/shared/ui/typography";
 
 import { useIssueCreationPanel } from "../../issue-creation-panel.context";
+import { MarkdownEditor } from "../markdown-editor/markdown-editor";
```

ESLint

[error] 'MarkdownEditor' is defined but never used.

(unused-imports/no-unused-imports)